osu.Game.Rulesets.Osu/Skinning/Legacy/OsuLegacySkinTransformer.cs

```diff
@@ -168,8 +169,23 @@ public OsuLegacySkinTransformer(ISkin skin)
 
                         return null;
 
-                    default:
-                        throw new UnsupportedSkinComponentException(lookup);
+                    case OsuSkinComponents.HitMarkerLeft:
```

Comment:
adding legacy sprites for new elements is generally not something we're doing.

Reply:
Would it be better to make these non-skinnable too?